### PR TITLE
Just get rid of all of the warnings

### PR DESCRIPTION
**Pull request by randomtestfive**

### CODE REVIEW

File: src/org/redalert1741/steamworks/Robot.java [hunk 1]
```diff
@@ -3,30 +3,23 @@
...
-import edu.wpi.first.wpilibj.XboxController;
-import edu.wpi.first.wpilibj.networktables.NetworkTable;
-import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 
 public class Robot extends IterativeRobot
 {
 	//private static LoggableNavX navx;
 	private static DataLogger logger;
 	private static Timer timer;
+	@SuppressWarnings("unused")
```
Comment (snydecor000) [inline src/org/redalert1741/steamworks/Robot.java:20]: wow this line is great

**Review by snydecor000 — DISMISSED** [review]
I dont like how you named robot "robot" plz fix

File: src/org/redalert1741/steamworks/Robot.java (hunk 1 above)
Comment (randomtestfive) [reply src/org/redalert1741/steamworks/Robot.java:20]: thank

Comment (dracco1993) [reply src/org/redalert1741/steamworks/Robot.java:20]: 11/10 would code again

Comment (randomtestfive) [reply src/org/redalert1741/steamworks/Robot.java:20]: perfect 5/7

Comment (tricktrap) [reply src/org/redalert1741/steamworks/Robot.java:20]: :rainbow: :+1:

**Review by dracco1993 — APPROVED** [review]

**Pull request merged**